src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -81,12 +83,46 @@ function StripePaymentForm({
 
     setIsLoading(true);
 
-    const { error } = await stripe.confirmPayment({
+    const stripePaymentMethodHandler = async (result) => {
+      if (result.error) {
+        // Show error in payment form
+        if (result.error.type === 'card_error' || result.error.type === 'validation_error') {
+          setMessage(result.error.message);
+        } else {
+          setMessage('An unexpected error occurred.');
+        }
+        setIsLoading(false);
+      } else {
+        // Otherwise send paymentIntent.id to your server
+        // TODO: refactor to fetch in service.js
+        const postData = formurlencoded({
+          payment_intent_id: result.paymentIntent.id,
+          first_name: firstName,
```

Comment:
Checking if we need these params here because `ecommerce` is retrieving this from Stripe anyways